linux/power_operation_script/set_power_script_custom.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   Set and check customized soft power operation
+# # Parameters:
+#   power_ops: power operation
+# # Return:
+#   script_file_path: path of customized script file
+#   script_tag_path: path of tag file produced by customized script file
+
+- name: "Set fact of customized script file"
+  ansible.builtin.set_fact:
+    script_file_folder: |-
+      {%- if guest_os_ansible_distribution == "Flatcar" -%}/var
+      {%- else -%}/tmp
+      {%- endif -%}
+    script_file_path: "{{ script_file_folder }}/{{ power_ops }}-vm-custom"
+    script_tag_path: "{{ script_file_folder }}/{{ power_ops }}.tag"
+
+- name: "Remove the existing script files for testcase {{ power_script_ops }}"
```

Comment:
Chang `script files` to `script and tag files`.
Change `testcase {{ power_script_ops }}` to `{{ power_script_op }} script`